repo/content/index.go

```diff
@@ -164,10 +164,6 @@ func (b *index) GetInfo(contentID ID) (*Info, error) {
 }
 
 func (b *index) entryToInfo(contentID ID, entryData []byte) (Info, error) {
-	if len(entryData) < 20 {
```

Comment:
Is this check performed inside `e.parse` ?

Reply:
Yes